code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -58,7 +58,9 @@
 		/obj/item/weapon/reagent_containers/glass,
 		/obj/item/weapon/storage/pill_bottle,
 		/obj/item/weapon/reagent_containers/pill,
-		/obj/item/weapon/reagent_containers/blood
+		/obj/item/weapon/reagent_containers/blood,
+		/obj/item/stack/material/phoron,
+		/obj/item/stack/medical
```

Comment:
They don't need to hold medical stacks.

Reply:
Is there a reason why medical grippers held by medical borgs shouldn't be able to pick up common medical items? Happy to remove it, just wondering why.

Reply:
Borgs, with some exceptions, have the tools they need. Giving them the ability to pick up anything else defeats the purpose of grippers.